### ISSUE TITLE
Renewed Vision ProPresenter 7 "Look" Controls

### ISSUE THREAD

**Issue by pyhasanon** [bitfocus/companion-module-requests#379]
Request update to Companion Module for Renewed Vision ProPresenter 7 to allow for controls to Switch "Audience Looks", or even a Dynamic control to show the currently activated "Look." Thank you.

**Comment by josephdadams** [comment]
@greyshirtguy

not possible with current Pro7 “api” right?
But you have some solutions you’re working on...

**Comment by greyshirtguy** [comment]
Companion uses the Remote mobile application protocol. (You know the app for iPhone and Android_....
We can only do whay they have added to that.  So far, RV has not added control for Looks to the remote app.. When they do - I will literaly trip over myself to start coding and add it to companion!

Side note: You may not be interested, but I have been experimenting with some hacks/workarounds and made some scripts/apps that let you control looks on the Pro7 machine itself.

MacOS
https://github.com/greyshirtguy/Pro7LooksHelper-AppleScript

Windows
https://github.com/greyshirtguy/Pro7LooksHelper-Win10UIA

**Comment by josephdadams** [comment]
Dan you should add teensy APIs to your Looks programs and then let the Companion module iterate from and control that. Everyone would love you for it, and I’m pretty sure you would get a sandwich named after you somewhere.

**Comment by pyhasanon** [comment]
> Companion uses the Remote mobile application protocol. (You know the app for iPhone and Android)....
> We can only do whay they have added to that. So far, RV has not added control for Looks to the remote app.. When they do - I will literaly trip over myself, running to my computer, to start coding and add it to companion!
> 
> Side note: You may not be interested, but I have been experimenting with some hacks/workarounds and made some scripts/apps that let you control looks on the Pro7 machine itself.
> 
> MacOS
> https://github.com/greyshirtguy/Pro7LooksHelper-AppleScript
> 
> Windows
> https://github.com/greyshirtguy/Pro7LooksHelper-Win10UIA

Yes, I actually found that earlier and played around with it.  Definitely convenient, but not as convenient as it would be via a Companion Module.  I appreciate the info, though!

**Comment by pyhasanon** [comment]
> Dan you should add teensy APIs to your Looks programs and then let the Companion module iterate from and control that. Everyone would love you for it, and I’m pretty sure you would get a sandwich named after you somewhere.

That would be amazing, and I vouch for the sandwich suggestion!

**Comment by greyshirtguy** [comment]
@joseph - I've been thinking about making a helper application for the Pro7 module to exactly that (and more).....
Might have to make a start soon!

**Comment by josephdadams** [comment]
Transferring to the pro7 repo.
